rack-mount/catalog/120mm-fan-tray_mod.scad

```diff
@@ -0,0 +1,61 @@
+use <../tray/tray.scad>
+
+/*
+  Simple tray for a beelink mini-s series mini pc.
+
+  Please make sure to configure the correct rack frame dimensions in rackFrame.scad.
+*/
+
+
+difference () {
+    bottomScrewTray (
+        u = 2,
+        trayWidth = 140,
+        trayDepth = 140,
+        trayThickness = 3,
+        frontLipHeight = 26,
+        backLipHeight = 6,
+        mountPoints = [[15, 15], [15+105, 15],[15, 15+105], [15+105, 15+105]],
+        frontThickness = 3,
+        sideThickness = 3,
+        mountPointElevation = 1,
+        mountPointType = "m4",
+        sideSupport = true,
+        trayLeftPadding = 15
+        );
+
+   /*Center Cube*/
+        translate(v=[67.5, 67.5, -50])
+            rotate(a=[90,0,0])
+            rotate(a=[0,90,90])
+            cylinder(h=60, r1=60, r2=60, $fn=75);
+           
+/*fan screw hole 1*/

```

Comment:
nit: indentation on the comments here (and other files in this pr) are inconsistent. Could you please go over the files and just make sure everything looks right? It's probably pretty inconsistent elsewhere though... I need to go over that at some point

Reply:
Went through each file to make indentations more uniformed.